docs/public-networks/reference/cli/subcommands.md

```diff
@@ -57,6 +57,8 @@ You can specify the starting index of the block range to import with `--start-bl
 
 You can specify the ending index (exclusive) of the block range to import with `--end-block`. If omitted, all blocks after the start block will be imported.
 
+You can specify multiple `--from` arguments. This can be useful when blocks have been exported over time to multiple files.
```

Comment:
Maybe clarify that the order of the block files will be the CLI order?

Reply:
Sure, that's a good point. Let me add that under a new commit.

Reply:
Added commit e1df688aa93603b83b8c224fb354b8d6e17338cc @shemnon